.changeset/smart-flies-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Previously added leniency in mouse precision when hovering from one `MenuItem` to its submenu now even works for `MenuItem`s that are not at the end of the menu.
```

Comment:
This could be worded in a way that describes the final outcome more clearly.

```suggestion
Submenus within a `DropdownMenu` will now consistently require less precision when moving the mouse between the parent item and the submenu.
```